### ISSUE TITLE
[BUG] Some issue with airdrop

### ISSUE THREAD

**Issue by helplanes** [TheBoredTeam/boring.notch#155]
**Airdrop window outline**
After dragging a file in the airdrop section, the window for airdrop does open, but then after that the files is not being sent to the receiver, the airdrop window closes with an outline of this window staying.

this outline stays after airdrop window closes, without the file being sent to the receiver.
<img width="736" alt="Screenshot 2024-10-22 at 6 17 15 PM" src="https://github.com/user-attachments/assets/bef014e2-77ac-4d16-9236-40de454ed31e">

**Comment by helplanes** [comment]
That window vanished after quitting the app

**Comment by richardkunkli** [comment]
Hi,
Thanks for the bug report. We will investigate it.

**Comment by helplanes** [comment]
Yes, its some issue with the airdrop feature itself i presume, cuz it keeps happening even after airdropping files now as well.
